sierra_adapter/common/src/test/scala/uk/ac/wellcome/sierra_adapter/utils/SierraAdapterHelpers.scala

```diff
@@ -1,89 +1,61 @@
-            s"Found record ${transformable.sierraId}, but VHS should be empty")
-      )
-      .map(_ => ())
+                 hybridStore: SierraVHS): hybridStore.WriteEither =
+    hybridStore.putLatest(transformable.sierraId.withoutCheckDigit)(
+      transformable)
 
   def storeInVHS(transformables: List[SierraTransformable],
-                 hybridStore: SierraVHS): Future[List[Unit]] =
-    Future.sequence(
-      transformables.map { t =>
-        storeInVHS(t, hybridStore = hybridStore)
-      }
-    )
-
-  def assertStored(transformable: SierraTransformable,
-                   table: Table): Assertion =
-    assertStored[SierraTransformable](
-      table = table,
-      id = transformable.sierraId.withoutCheckDigit,
-      record = transformable
-    )
-
-  def assertStoredAndSent[T](t: T, id: String, topic: Topic, table: Table)(
-    implicit decoder: Decoder[T]): Assertion = {
-    val hybridRecord = getHybridRecord(table, id = id)
-
-    val storedTransformable = getObjectFromS3[T](
-      bucket = Bucket(hybridRecord.location.namespace),
-      key = hybridRecord.location.key
-    )
-    storedTransformable shouldBe t
-
-    listMessagesReceivedFromSNS(topic).map { info: MessageInfo =>
-      fromJson[HybridRecord](info.message).get
-    } should contain(hybridRecord)
-  }
-
-  def assertStoredAndSent(transformable: SierraTransformable,
-                          topic: Topic,
-                          table: Table): Assertion =
-    assertStoredAndSent[SierraTransformable](
-      transformable,
-      id = transformable.sierraId.withoutCheckDigit,
-      topic = topic,
-      table = table
-    )
+                 hybridStore: SierraVHS): Seq[hybridStore.WriteEither] =
+    transformables.map { t =>
+      storeInVHS(t, hybridStore = hybridStore)
+    }
+//  def assertStored(transformable: SierraTransformable,
+//                   table: Table): Assertion =
+//    assertStored[SierraTransformable](
+//      table = table,
+//      id = transformable.sierraId.withoutCheckDigit,
+//      record = transformable
+//    )
+//
+//  def assertStoredAndSent[T](t: T, id: String, topic: Topic, table: Table)(
+//    implicit decoder: Decoder[T]): Assertion = {
+//    val hybridRecord = getHybridRecord(table, id = id)
+//
+//    val storedTransformable = getObjectFromS3[T](
+//      bucket = Bucket(hybridRecord.location.namespace),
+//      key = hybridRecord.location.key
+//    )
+//    storedTransformable shouldBe t
+//
+//    listMessagesReceivedFromSNS(topic).map { info: MessageInfo =>
+//      fromJson[HybridRecord](info.message).get
+//    } should contain(hybridRecord)
+//  }
+//
+//  def assertStoredAndSent(transformable: SierraTransformable,
+//                          topic: Topic,
+//                          table: Table): Assertion =
+//    assertStoredAndSent[SierraTransformable](
+//      transformable,
+//      id = transformable.sierraId.withoutCheckDigit,
+//      topic = topic,
+//      table = table
+//    )
```

Comment:
Are you going to use these assertion helpers again? If not, can they be deleted rather than just commented out?